enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/analytics/AnalyticsConstants.java

```diff
@@ -25,8 +25,17 @@ public class AnalyticsConstants {
 
     public static final String PUBLISHER_IMPL_CONFIG_KEY = "publisherImpl";
     protected static final String IS_CHOREO_DEPLOYMENT_CONFIG_KEY = "isChoreoDeployment";
+    protected static final String TYPE_CONFIG_KEY = "type";
```

Comment:
These now configs should go to config.toml

Reply:
Added to the config.toml.template file as analytics are disabled by default in the config.toml